### ISSUE TITLE
`minCellCount` suppression in `time_distribution.csv`

### ISSUE THREAD

**Issue by msuchard** [OHDSI/CohortDiagnostics#865]
`minCellCount` suppression does not appear to applied to results in `time_distribution.csv`.   This may have been fixed in a more recent version, but needs double-check nonetheless.

**Comment by gowthamrao** [comment]
@azimov 

time_distribution is now (as of version 3) calculated using the featureExtraction call with covariateRef = temporalCovariateRef. However the export of the output does not use the new makeDataExportable function. It has its own function

https://github.com/OHDSI/CohortDiagnostics/blob/main/R/ExportCharacterization.R

This function does use minCellCount and it appears (per code review only) that it should have enforced the minCellCount.

**Comment by gowthamrao** [comment]
Also this change fixed a previously reported bug i thought
https://github.com/OHDSI/CohortDiagnostics/commit/c42bc7d117543fa9e1bdf757aead64b98ab5eea2

discussed here https://github.com/OHDSI/CohortDiagnostics/issues/658

**Comment by azimov** [comment]
@msuchard I'm curious which version this code was executed on. I have made a new PR that uses a common export function that checks defined columns for minimum values (however, I was confident that the values exported were filtered before this).

See pull request #867 for full refactoring

**Comment by msuchard** [comment]
`LegendT2dm` uses `v2.2.3` and `v2.1.3` .   There does not appear to be any censoring in `R/TimeDistributions.R`.   If, however, this is fixed (per @azimov above) in newer versions, then I guess we are OK.

**Comment by azimov** [comment]
Closing this issue I have merged #867 in to develop, however v2.1.3 (and possibly 2.2.x as well) had a very different method for ensuring values were censored. 


@msuchard I'm confident that version 3.0.x+ should not have this issue and certain that the next release will resolve it. For legacy studies there may be a solution that can be implemented in the study package if this is needed?

I also think more testing around each individual diagnostic is required to ensure this remains the case going forwards. In this case I added unit tests for the temporal/Characterization results but there is significant room for improvement.
